lib/ansible/cli/galaxy.py

```diff
@@ -1485,7 +1485,7 @@ def execute_list_collection(self, artifacts_manager=None):
                 if os.path.isdir(collection_path):
                     display.vvv("Searching {0} for collections".format(collection_path))
                     collections = list(find_existing_collections(
-                        collection_path, artifacts_manager,
+                        collection_path, artifacts_manager, require_mandatory_keys=False
```

Comment:
Mind keeping the trailing comma?
```suggestion
                        collection_path, artifacts_manager, require_mandatory_keys=False,
```

Reply:
Rather than thread this through the entirety of find_existing_collections, I thought it might be easier to read if I hang it on the ConcreteArtifactManager. Now it's a property that execute_list_collection sets.